content/platforms/release-notes/k8s-6-0-8-20-2020-12.md

```diff
@@ -0,0 +1,93 @@
+---
+Title: Redis Enterprise for Kubernetes Release Notes 6.0.8-20 (December 2020)
+description: Release notes for version 6.0.8-20 (latest)
+weight: 72
+alwaysopen: false
+categories: ["Platforms"]
+---
+
+The Redis Enterprise K8s 6.0.8-20 release is a *major release* on top of [6.0.8-1](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases/tag/v6.0.8-1) providing support for the latest [Redis Enterprise Software release 6.0.8-30](https://docs.redislabs.com/latest/rs/release-notes/rs-6-0-8-september-2020/) and includes several enhancements  and bug fixes.
+
+## Overview
+
+This release of the operator provides:
+
+ * The latest release version of the operator
+ * New features
+ * Various bug fixes
+
+
+Follow the instructions in ["Upgrading a Redis Enterprise Cluster in Operator-based Architecture"]({{< relref "/platforms/kubernetes/upgrading-kubernetes-with-operator.md" >}}) to upgrade to this Kubernetes operator release.
+
+## Images
+
+ * **Redis Enterprise** - redislabs/redis:6.0.8-30 or redislabs/redis:6.0.8-30.rhel7-openshift
+ * **Operator** - redislabs/operator:6.0.8-20
+ * **Services Rigger** - redislabs/k8s-controller:6.0.8-20 or redislabs/services-manager:6.0.8-20 (Red Hat registry)
+
+## New features
+
+ * Databases custom resources (REDB) can be created in separate consumer namespaces from the operator and cluster namespace. The operator deployment is configured to watch specific namespaces for these database REDB CR’s.
```

Comment:
```suggestion
 * You can now create database custom resources (REDB CRs) in consumer namespaces that are separate from the operator and cluster namespace. You configured the operator deployment to watch specific namespaces for these REDB CRs.
```

Reply:
@alexmilowski Please write in the active voice. Also, please confirm that I have not changed the meaning here.